src/components/modals/UsersSelectModalHub.js

```diff
@@ -60,12 +78,14 @@ export default function UsersSelectModalHub({ type }) {
     return chatName ? (
       <UserSelectorBlock
         closeWindow={closeModal}
+        isEncrypted={isEncrypted}
```

Comment:
should we also add `isEncrypted` to deps of `const typeOfFunc = useMemo(() => {` ?